server/command.go

```diff
@@ -238,6 +239,23 @@ func (p *Plugin) ExecuteCommand(_ *plugin.Context, args *model.CommandArgs) (*mo
 		response := p.getCommandResponse(args, message)
 		return response, nil
 
+	case commandBuild:
+		if len(parameters) != 2 {
+			return p.getCommandResponse(args, "Command arguments mismatched, please use `/gitlab help` to see action parameters. "), nil
+		}
+		project := parameters[0]
+		ref := parameters[1]
```

Comment:
Should the command  default to default branch?

Reply:
my bad

Reply:
fixed by https://github.com/mattermost/mattermost-plugin-gitlab/pull/280/commits/796f2286fe8580ec4dc31f98aef9de01e745f3aa